systems/docker-compose.yml

```diff
@@ -143,11 +167,14 @@ services:
       - DB_PASSWORD=ThisIsUkamaXPass
       - DB_USER=postgres
       - QUEUE_URI=amqp://guest:guest@rabbitmq:5672
+      - MSGCLIENT_HOST=msg-client-registry:9095
+      - ORG_SERVICE_HOST=org
+      - ORG_SERVICE_PORT=9090
       - UKAMA_UUID=${UKAMA_UUID}
```

Comment:
I think we dont need that environment variable anymore.

Reply:
We need `MSGCLIENT_HOST`, `ORG_SERVICE_HOST ` and `ORG_SERVICE_HOST ` Env's to connecting with msgClient. But yes, you're right about `UKAMA_UUID` i'll remove it.

Reply:
addressed